regression/scripts/aerodynamics_super/aerodynamics_super.py

```diff
@@ -301,7 +301,7 @@ def main():
     
     ## Only one wing is evaluated since they rely on the same function
     #cd_c           = drag_breakdown.compressible['main_wing'].compressibility_drag
-    #cd_i           = drag_breakdown.induced.total
+    #cd_i           = 
```

Comment:
Do these comments need to stay here?

Reply:
I will remove this file as it is no longer used/ a part of the regression